tests/run-make/profile/rmake.rs

```diff
@@ -0,0 +1,27 @@
+// This test revolves around the rustc flag -Z profile, which should
+// generate a .gcno file (initial profiling information) as well
+// as a .gcda file (branch counters). The path where these are emitted
+// should also be configurable with -Z profile-emit. This test checks
+// that the files are produced, and then that the latter flag is respected.
+// See https://github.com/rust-lang/rust/pull/42433
+
+//@ ignore-cross-compile
```

Comment:
The CI failure is caused by a missing `//@ needs-profiler-support` here :) The `-Cinstrument-coverage` flag isn't relevant here.